onnx/helper.py

```diff
@@ -19,6 +19,7 @@
 import google.protobuf.message
 import numpy as np
 import typing_extensions
+from google.protobuf.internal.containers import RepeatedCompositeFieldContainer
```

Comment:
You may hide this under the TYPE_CHECKING block I think

Reply:
Can